docs/content/staking/index.mdx

```diff
@@ -1,22 +1,25 @@
 ---
 title: Epochs, Staking & Delegating on Flow
 sidebar_title: Introduction
-description: Introduction to how staking works on Flow
+description: 
 ---
 
-This document provides an introduction to staking FLOW tokens on the Flow network for token holders and node operators.
+This document provides an introduction to staking FLOW tokens on the Flow network
+for token holders and node operators.
 Staking is an important part of the security protocol of a proof-of-stake (PoS) blockchain.
-Running nodes and staking tokens contributes to the blockchain's security and is rewarded accordingly.
+Running nodes and staking tokens contributes to the blockchain's
+security and is rewarded accordingly.
 
 ## What is Staking?
 
 Flow is a global network of computers working together
 to maintain the security and integrity of its users' data.
 
 This global network is made up of many individual nodes: software applications run by people.
-Every node in the network shares a small part of the responsibility to keep the network running smoothly
-and to ensure that other nodes are doing the same. This shared responsibility is sometimes called decentralization,
-because no single central node is solely responsible for the security and integrity of the network and the data it contains.
+Every node in the network shares a small part of the responsibility
+to keep the network running smoothly and to ensure that other nodes are doing the same.
+This shared responsibility is sometimes called decentralization, because no single central
```

Comment:
```suggestion
This shared responsibility is the core premise of decentralization, because no single central
```

'sometimes called' feels too wooly and vague. Decentralization is either shared ownership, or else it ain't and thus it's centralized, right?